src/tests/Cns.test.ts

```diff
@@ -605,4 +606,137 @@ describe('CNS', () => {
       ).rejects.toEqual(new Error('error_up'));
     });
   });
+
+  describe('.tokenURI', () => {
+    it('should return token URI', async () => {
+      const spies = mockAsyncMethods(cns.readerContract, {
```

Comment:
`mockAsynMethods` is ignored when running live tests. It worth checking tests cases with live tests run. You might need to use existing domain instead of test.crypto. For example: ryan.crypto

Reply:
Code reference: `src/tests/helpers.ts:41`